packages/core/src/Core.ts

```diff
@@ -119,26 +118,26 @@ export class Core<T extends JWPluginConfig = JWPluginConfig> extends JWPlugin<T>
         if (range.isCollapsed()) {
             // Basic case: remove the node directly following the range.
             const nextSibling = range.end.nextSibling();
-            if (nextSibling && this.mode.is(nextSibling, RuleProperty.EDITABLE)) {
+            if (nextSibling && this.editor.mode.is(nextSibling, RuleProperty.EDITABLE)) {
```

Comment:
Didn't we say that the mode should be taken from the range in context rather than the editor ?

Reply:
Damn you're absolutely right... :O